packages/aperf/aperf.0.1.1/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+maintainer: "Philip Dexter <philip.dexter@gmail.com>"
+authors: "Philip Dexter <philip.dexter@gmail.com>"
+homepage: "http://github.com/philipdexter/aperf"
+bug-reports: "http://github.com/philipdexter/aperf/issues"
+dev-repo: "http://git.phfilip.com/aperf"
```

Comment:
No, the url does not have a `.git` at the end. It seems to be valid in opam 1.2.2 just not 1.2.0. Maybe I'll just constrain opam to 1.2.2.